### DISCUSSION TITLE
Error when turning on Vue DevTools

### DISCUSSION THREAD

**Discussion by Webess** [laravel/nova-issues#3489] [Q&A]
- Laravel Version: 8.40
- Nova Version: 3.27.0
- PHP Version: 8.0.2
- Database Driver & Version: MySQL 5.7.24
- Operating System and Version: Windows 10 Pro 2004
- Browser type and version: Chrome 

### Description:
I try to turn on Vue DevTools according to the instructions in the documentation, but after running the command `npm run dev` it always stuck at 10% for a few minutes and then crashes. This is the same out of memory error as in #2901.

npm log: [2021-08-03T08_43_19_011Z-debug.log](https://github.com/laravel/nova-issues/files/6922829/2021-08-03T08_43_19_011Z-debug.log)
console log (after running npm run dev): [console.log](https://github.com/laravel/nova-issues/files/6922850/console.log)

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. cd ./vendor/laravel/nova
2. mv webpack.mix.js.dist webpack.mix.js
3. npm install
4. npm run dev
5. rm -rf node_modules
6. cd -
7. php artisan nova:publish

**Comment by crynobone** [comment]
Seems like an NPM space issue based on quick search.

```
FATAL ERROR: MarkCompactCollector: young object promotion failed Allocation failed - JavaScript heap out of memory
```

You could try modifying the default npm config or use `yarn` instead

**Reply by Webess** [reply]
Thank you for your response. Using `yarn` resulted in the same problem. This seems to be a problem when using cmd on Windows. When I ran it under WSL, it went smoothly.
